app/forms/providers/application_merits_task/plf_court_order_form.rb

```diff
@@ -6,6 +6,10 @@ class PLFCourtOrderForm < BaseForm
       attr_accessor :plf_court_order
 
       validates :plf_court_order, inclusion: %w[true false], unless: :draft?
+
+      def copy_of_court_order?
+        plf_court_order == "true"
```

Comment:
I'm interpreting this ticket as:
if their current answer is `no` and then they change their answer to `yes` they'll see evidence upload, but if their current answer is `yes` and they still choose `yes` they'll just go to CYA

They way this seems to work is:
if their current answer is `yes` and they still choose `yes`, they go to evidence upload

Not a major issue to me either way, but just flagging